### PR TITLE
Add Laravel Zero and PHP Monitor

### PR DESCRIPTION
**Pull request by RobertBoes**
Adds [Laravel Zero](https://github.com/laravel-zero/laravel-zero) and [PHP Monitor](https://github.com/nicoverbruggen/phpmon)

### CODE REVIEW

File: database/packages.ts [hunk 1]
```diff
@@ -5431,6 +5448,23 @@ export const laravelPackages: Package[] = [
         php_only: false,
         updated_at: '2023-04-04T14:16:54.316Z',
     },
+    {
+        name: 'PHP Monitor',
+        description: 'Native Mac menu bar app that interacts with Laravel Valet. Helps you manage multiple PHP installations, locate config files and more.',
+        category: 'Utilities & Helpers',
```
Comment (innocenzi) [inline database/packages.ts:5454]: If think this should be devtools

File: database/packages.ts [hunk 2]
```diff
@@ -5431,6 +5448,23 @@ export const laravelPackages: Package[] = [
         php_only: false,
         updated_at: '2023-04-04T14:16:54.316Z',
     },
+    {
+        name: 'PHP Monitor',
+        description: 'Native Mac menu bar app that interacts with Laravel Valet. Helps you manage multiple PHP installations, locate config files and more.',
+        category: 'Debugging & Dev Tools',
+        github: 'https://github.com/nicoverbruggen/phpmon',
+        author: 'Nico Verbruggen',
```
Comment (RobertBoes) [inline database/packages.ts:5456]:
```suggestion
        author: 'nicoverbruggen',
```

File: database/packages.ts [hunk 3]
```diff
@@ -1225,6 +1225,23 @@ export const laravelPackages: Package[] = [
         php_only: false,
         updated_at: '2023-04-04T14:09:48.903Z',
     },
+    {
+        name: 'Laravel Zero',
+        description: 'A micro-framework built on top of Laravel components for building command-line applications',
+        category: 'Code Architecture',
+        github: 'https://github.com/laravel-zero/laravel-zero',
+        author: 'Nuno Maduro and Owen Voke',
```
Comment (RobertBoes) [inline database/packages.ts:1233]: Not sure what the author should be here, I just noticed they're github usernames (I think?), but this package has two authors mentioned in their readme

Comment (HassanZahirnia) [reply database/packages.ts:1233]: The `author` should be the name of the composer package owner, So if it's `aimeos/aimeos-laravel` it becomes the first part before the slash: `aimeos`

Comment (HassanZahirnia) [reply database/packages.ts:1233]: In some cases where there is no composer package, and it's a npm package (which does not indicate an owner from my understanding), you can use their github username in lowercase, or something they're famous for.
Like `https://github.com/xiCO2k/laravel-vue-i18n` which is a npm (`laravel-vue-i18n`) package the author becomes `xico2k`

Comment (HassanZahirnia) [reply database/packages.ts:1233]: The idea is the `author` has to be unique so we can use it for filters and search and some other stuff

**Pull request merged**